### PR TITLE
new add rock5b power

### PR DESCRIPTION
**Pull request by wallace77777**
###### Review guide

To review this PR locally, please run the following commands:

```bash
PR= # the current PR number
git fetch origin pull/${PR}/head
git switch --detach FETCH_HEAD
yarn start
yarn start --locale en
# If you want to start 2 locales side-by-side, use the following command on Linux:
# https://github.com/facebook/docusaurus/issues/7377#issuecomment-1167192715
yarn start & DOCUSAURUS_GENERATED_FILES_DIR_NAME=.docusaurus/en yarn start --locale en --port 3001
kill %1
```

### CODE REVIEW

**Comment by RadxaYuntian** [comment]
改了一下。@wallace77777 @Maycides @xzuoqi 麻烦审核一下

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/rock5/rock5b/getting-started/power.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+sidebar_label: 'Power Management'
 sidebar_position: 18
 ---
 
-# Powen on/off
+# Power Management
+
+## Control with hardware power key
+
+When the ROCK 5B is connected to the power supply, the green power LED will stay on. Press the power button to start the system.
```
Comment (Maycides) [inline i18n/en/docusaurus-plugin-content-docs/current/rock5/rock5b/getting-started/power.md:10]:
```suggestion
When the ROCK 5B is connected to the power supply and it's shut down, the green power LED will stay on. Press the power button to start the system.
```

**Comment by wallace77777** [comment]
> 改了一下。@wallace77777 @Maycides @xzuoqi 麻烦

File: docs/rock5/rock5b/getting-started/power.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+sidebar_label: '电源管理'
 sidebar_position: 18
 ---
 
-# Coming Soon
+# 电源管理
+
+## 使用硬件电源键
+
+当 ROCK 5B 接上电源之后，电源绿灯将会一直常亮。按下电源键即可启动。
```
Comment (RadxaYuntian) [inline docs/rock5/rock5b/getting-started/power.md:10]:
```suggestion
当 ROCK 5B 接上电源之后，在关机状态下电源绿灯将会一直常亮。此时按下电源键即可启动。
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Comment by RadxaYuntian** [comment]
@wallace77777 @Maycides @xzuoqi 没问题的话在 Files changed 里面approve一下

**Review by Maycides — APPROVED** [review]

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**